restcomm/restcomm.application/src/main/webapp/WEB-INF/conf/restcomm.xml

```diff
@@ -266,6 +266,9 @@
 			 or send appropriate response according to the response from outbound dial-->
 		<!--  https://github.com/RestComm/Restcomm-Connect/issues/1649 -->
 		<enable-200-ok-delay>false</enable-200-ok-delay>
+        
+                <!-- Maximum P2P call length within RestComm, unit is minute -->
```

Comment:
do you expect this new configuration property to be configured for Docker container?. I think we are missing changes to autoconfig scripts there... not sure if you are planning to provide those later in appropiate repo

Reply:
@jaimecasero 
This configuration is for specific customer, we don't face with this requirement on cloud, so I'm not expecting to have automatic script do any job.

Reply:
i think is still required to add a new property here https://github.com/RestComm/Restcomm-Connect/blob/master/restcomm/configuration/config-scripts/as7-config-scripts/restcomm/advanced.conf

and then deliver this into the actual xml by  adding some scripting here 
https://github.com/RestComm/Restcomm-Connect/blob/master/restcomm/configuration/config-scripts/as7-config-scripts/restcomm/autoconfig.d/config-restcomm.sh#L718